motionplan/tpSpaceRRT.go

```diff
@@ -718,9 +723,9 @@ func (mp *tpSpaceRRTMotionPlanner) extendMap(
 	for _, newNode := range newNodes {
 		ptgNum := int(newNode.Q()[0].Value)
 		randAlpha := newNode.Q()[1].Value
-		randDist := newNode.Q()[2].Value
+		randDist := newNode.Q()[3].Value - newNode.Q()[2].Value
 
-		trajK, err := mp.tpFrame.PTGSolvers()[ptgNum].Trajectory(randAlpha, randDist, mp.planOpts.Resolution)
+		trajK, err := mp.tpFrame.PTGSolvers()[ptgNum].Trajectory(randAlpha, 0, randDist, mp.planOpts.Resolution/5)
```

Comment:
why do we make `Resolution` 5 times greater here, assuming `Resolution < 1.0`

Reply:
I think this was left over from different testing, will remove